src/components/Modal/Modal.driver.ts

```diff
@@ -6,22 +6,23 @@ import { UniDriver } from 'wix-ui-test-utils/unidriver';
 import { MODAL_DATA_HOOKS } from './dataHooks';
 
 export interface ModalDriver extends BaseUniDriver {
-  getModalStage(): UniDriver;
+  getModalContent(): UniDriver;
   isModalShowed(): Promise<boolean>;
```

Comment:
I know it's not part of your code, but maybe you can change `isModalShowed` to `isShown` or just `isOpen` 😬 ?

Reply:
sure